pkg/catalog/manager/traverse.go

```diff
@@ -136,7 +145,13 @@ func (m *Manager) traverseAndUpdate(repoPath, commit string, catalog *v3.Catalog
 				}
 			}
 
-			v.ExternalID = fmt.Sprintf("catalog://?catalog=%s&template=%s&version=%s", catalog.Name, template.Spec.FolderName, v.Version)
+			if catalogType == client.CatalogType {
+				v.ExternalID = fmt.Sprintf("catalog://?catalog=%s&template=%s&version=%s", catalog.Name, template.Spec.FolderName, v.Version)
```

Comment:
changing format might break other things. We have other places parsing this format.